inst/odin/carehomes.R

```diff
@@ -428,7 +428,10 @@ I_with_diff_trans[, ] <-
     comm_D_transmission * sum(I_comm_D[i, , j]))
 
 s_ij[, ] <- m[i, j] * sum(I_with_diff_trans[j, ])
-lambda[] <- beta * sum(s_ij[i, ])
+## beta applied to all contacts except within care home contacts
+s_ij[1:(N_age - 2), 1:N_age] <- beta * s_ij[i, j]
+s_ij[(N_age - 1):N_age, 1:(N_age - 2)] <- beta * s_ij[i, j]
```

Comment:
Maybe change these to 1:15 and 16:17 (as we do for serology groups below), 
also line 433 still seems to contain beta?

Reply:
We just don't apply to the 2 x 2 submatrix in the corner. Is it clearer now?